src/components-examples/journey-maps/angular/journey-maps-routes-zones/journey-maps-routes-zones-example.html

```diff
@@ -6,7 +6,7 @@
     [apiKey]="apiKey"
     language="en"
     #advancedMap
-    [journeyMapsRoutingOption]="journeyMapsRoutingOption"
+    [journeyRoutesOption]="journeyRoutesOption"
```

Comment:
This would be a breaking change, and AFAIK we are trying to implement ROKAS-2171 without any breaking changes.